game-core/src/test/java/games/strategy/engine/chat/ChatPlayerPanelTest.java

```diff
@@ -0,0 +1,31 @@
+package games.strategy.engine.chat;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+import java.awt.Dimension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class ChatPlayerPanelTest {
+
+    @Mock private PlayerChatRenderer renderer;
+
+    private ChatPlayerPanel chatPlayerPanel;
```

Comment:
Codacy found an issue: [Perhaps 'chatPlayerPanel' could be replaced by a local variable.](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5821454)